uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-task.controller.js

```diff
@@ -258,6 +259,18 @@ define([
 
       return true;
     }
+
+    /**
+     * Opens editor for activity type options editing
+     */
+    function openActivityTypeOptionsEditor () {
```

Comment:
This is a duplicate code with https://github.com/compucorp/civihr-tasks-assignments/pull/360/files#diff-76ee16aaa0874056d972e19b2f08802fR605, did you have any thoughts on how to avoid that?

Reply:
Not really as i think the two of them belongs to different scope.

Reply:
It's fine actually, I looked at it with a fresh eye and despite of the fact the functions look the same and do the same I don't think we should move them to a separate service of something, that may be an overkill.